mirgecom/viscous.py

```diff
@@ -103,8 +104,9 @@ def viscous_stress_tensor(state, grad_cv):
 
 # low level routine works with numpy arrays and can be tested without
 # a full grid + fluid state, etc
-def _compute_diffusive_flux(density, d_alpha, grad_y):
-    return -density*d_alpha.reshape(-1, 1)*grad_y
+def _compute_diffusive_flux(density, d_alpha, y, grad_y):
```

Comment:
Just add a comment that the second line is the correction velocity.

Reply:
[6c706d2](https://github.com/illinois-ceesd/mirgecom/pull/868/commits/6c706d2f1a0fb7f5ef27dba1b9496e2c87de84fb)